clients/aws-sdk-bedrock-runtime/docs/README.md

```diff
@@ -4,7 +4,7 @@ Sphinx is used for documentation. You can generate HTML locally with the
 following:
 
 ```
-$ uv pip install ".[docs]"
+$ uv pip install --group docs .
```

Comment:
I made this change manually since the install command is wrong. Here is the permanent fix:

https://github.com/smithy-lang/smithy-python/pull/563